frontend/src/scenes/PreflightCheck/PreflightCheck.scss

```diff
@@ -0,0 +1,176 @@
+@import '~/vars';
+
+.preflight-check-container {
+    min-height: 100vh;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: row;
+    padding: 40px 16px;
+
+    .preflight-box {
+        @extend %mixin-elevated;
+        max-width: 480px;
+        background-color: var(--bg-light);
+        border-radius: $radius;
+        padding: 40px;
+
+        .divider {
+            margin: 24px;
+            border-top: 2px dashed var(--border);
+        }
```

Comment:
Do we plan to use this divider style in places other than the preflight check panel? @lottiecoxon If so, it might be worth making this style a mode of `LemonDivider`. Doesn't make sense if it's just one-off!

Reply:
I've seen this line in a few other onboarding mocks, so I just added the `thick` and `dashed` prop to `LemonDivider`